### DISCUSSION TITLE
A minimize button in the native OS option bar.

### DISCUSSION THREAD

**Discussion by TacoGuy845** [elementary/triage#503] [Ideas]
Hello, I recently joined to support the development of elementary, so I don't know if it is a design decision, but is there no option to minimize active apps in the OS's native options bar? I don't know if I'm the only one, but it really makes me anxious not to have that within reach heh.
It would be great to have all 3 options together (minimize, restore/maximize, and close) either on the right or on the left.

Greetings to the development team, the new look of elementary looks amazing 🔥

**Comment by danirabbit** [comment]
Pantheon is workspace based, so the way to temporarily background apps when multitasking is to place them on a different workspace using either keyboard shortcuts, gestures, hot corners, or the multitasking view icon in the dock

**Reply by jlnr** [reply]
But it does have the cmd+H shortcut and I always end up using that a lot. I hope I'm not jinxing this shortcut into being removed, but given that it exists, why not have an optional toolbar icon for mouse users to match? Of course it would be "Hide" and not "Minimize" for consistency. I personally don't need it but it seems like the most common request (maybe tied with desktop icons).

**Reply by TacoGuy845** [reply]
oh, I see, thanks for the clarifications and recommendations, I will put them into practice 👾

**Comment by alvaromellado** [comment]
Alternatively if you don't like it, You can assign the minimize action to shortcut (For example: My mouse has an action button that I never use, I assign it to minimize action)

**Reply by TacoGuy845** [reply]
woah, I hadn't thought about it, but it's a good idea, thx

**Comment by 1280px** [comment]
See also https://github.com/elementary/switchboard-plug-pantheon-shell/issues/308

If you want to re-add it via 3rd-party tools: https://github.com/pantheon-tweaks/pantheon-tweaks
